cast/src/lib.rs

```diff
@@ -447,8 +453,40 @@ impl SimpleCast {
         format!("0x{}", s)
     }
 
+    pub async fn generate_interface(
+        address_or_path: String,
+        chain: Chain,
+        etherscan_api_key: Option<String>,
+    ) -> Result<Vec<InterfaceSource>> {
+        let mut contract_names: Vec<String> = Vec::new();
+        let contract_abis: Vec<Abi> = if Path::new(&address_or_path).exists() {
+            println!("Reading ABI from local file at {}", &address_or_path);
```

Comment:
remove the printlns, they're not necessary IMO and they make piping the stdout to a file harder